yt/yt/tests/integration/formats/test_skiff_format.py

```diff
@@ -823,8 +853,8 @@ def test_skiff_decimal_inside_composite(self, precision, binary_size, skiff_type
             )
         )
         skiff_data = (
-            b"\x00\x00\x3a\x01" + b"\x00" * (binary_size - 2) + b"\x02\x00\x00\x00" b"pi"
-            + b"\x00\x00\xf1\xfe" + b"\xff" * (binary_size - 2) + b"\x07\x00\x00\x00" b"minus e"
+            self._encode_row(self._get_encoded_decimal_pi(binary_size, skiff_type) + self._encode_string32(b"pi"))
```

Comment:
Without new code in skiff yson converter this error occurs here:

```bash
E   yt.common.YtResponseError: Received response with error
E       Cannot create Skiff parser for table #0
E           Cannot create Skiff parser for column "column"
E               Skiff type yson32 cannot represent type Decimal<35, ?>
```

Reply:
Overall looks good to me, but I would like to make a few improvements in tests internally.

Reply:
Shipped!